build_tools/python/e2e_test_artifacts/cmake_generator/common_generators.py

```diff
@@ -0,0 +1,55 @@
+## Copyright 2022 The IREE Authors
```

Comment:
Similar to #10901 , maybe rename this to `model_generators` ?

Reply:
Renamed to `model_rule_generator`.